pages/linux/gdc.md

```diff
@@ -0,0 +1,24 @@
+# gdc
+
+> GNU front-end D Compiler.
+> More information: <https://wiki.dlang.org/Using_GDC>.
+
+- Create an executable:
+
+`gdc {{Source_Name}}.d -o {{Output_File_Name}}`
```

Comment:
Please use the path syntax [suggested in our style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths)

```suggestion
`gdc {{path/to/source.d}} -o {{path/to/output_executable}}`
```